lambdas/credentialissuerreturn/src/main/java/uk/gov/di/ipv/core/credentialissuerreturn/CredentialIssuerReturnHandler.java

```diff
@@ -90,8 +91,22 @@ public APIGatewayProxyResponseEvent handleRequest(
             APIGatewayProxyRequestEvent input, Context context) {
         LogHelper.attachComponentIdToLogs();
         try {
-            auditService.sendAuditEvent(AuditEventTypes.IPV_CRI_AUTH_RESPONSE_RECEIVED);
-            RequestHelper.getIpvSessionId(input);
+
+            String ipvSessionId = RequestHelper.getIpvSessionId(input);
+            String userId = ipvSessionService.getUserId(ipvSessionId);
+
+            AuditEventUser auditEventUser = new AuditEventUser(userId, ipvSessionId);
+            this.componentId =
+                    configurationService.getSsmParameter(
+                            ConfigurationVariable.AUDIENCE_FOR_CLIENTS);
```

Comment:
In the log helper we're setting the componentID to a hard coded value of `core`. That way it's the same between all envs, and the same Splunk query will work between envs. If you use the audience value, you're gonna get different values per env. Either way I think the two things should match.

Reply:
There's been a bit of discussion about what the componentId should be. It's primarily the name for the id we use to represent the core when talking to other services, and used in aud and iss in jwts. Really parameter in config should be `self/componentId` and we should use that as appropriate where when creating and validating jwts.

Reply:
The log helper should reflect that too.